terraform/install.sh

```diff
@@ -229,6 +229,9 @@ installMonitoring() {
     read -p "${YELLOW}When you are done, please PRESS ENTER TO CONTINUE"
   fi
 
+  log "Checking to make sure necessary Istio services are ready for monitoring"
+  python3 -m pip install google-cloud-monitoring
```

Comment:
This should probably be installed as part of our base image (@eaberman)

Reply:
Good point! I'll open up an issue for this and address it.